src/components/footer/footer-bar.vue

```diff
@@ -44,6 +45,15 @@ function onClickLayersIcon() {
 
 watch(drawToolbarOpen, isOpen => isOpen && (measureToolbarOpen.value = false))
 watch(measureToolbarOpen, isOpen => isOpen && (drawToolbarOpen.value = false))
+
+const featureInfoStore = useFeatureInfoStore()
+const { infoPanelHidden } = storeToRefs(featureInfoStore)
+const { setInfoPanelHidden } = featureInfoStore
```

Comment:
Any reason why this in featureInfoStore instead of the app store? (like legendsOpen, feedbackOpen, ...). Also this naming is about if the panel is hidden whereas for all the other, it is about the opening state (not very important but maybe we should stick to the same logic for the naming?)

Reply:
I think the naming is not ideal here. The `infoPanelHidden` is actually something different than the `infoOpen` which also exists in the `app.store`. The `infoPanelHidden` is only used to check on mobile if the star should be displayed (since the panel is hidden, but a feature info is present). I've renamed it into `displayStarOnMobile` to make things clearer.